app/Http/Controllers/QuestionController.php

```diff
@@ -0,0 +1,134 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Validator;
+use App\Question;
+use App\QuestionOption;
+use Illuminate\Http\Request;
+use App\Http\Resources\Question as QuestionResource;
+
+class QuestionController extends Controller
```

Comment:
Rename the controller to OneChoiceQuestionController, as far as you should also add controllers for the other question types.